presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -96,6 +96,14 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: last_day(date) -> date
```

Comment:
The function should only be listed once in the documentation. The parameters are names, not types, so we only list multiple versions if the argument count is different. For example, we have four versions of the `month()` function, but it's only listed once.